metadata.yaml

```diff
@@ -10,16 +10,24 @@ issues: https://github.com/canonical/sdcore-nms-operator/issues
 containers:
   nms:
     resource: nms-image
+    mounts:
+      - storage: config
+        location: /nms/config
 
 resources:
   nms-image:
     type: oci-image
     description: OCI image for SD-Core NMS
-    upstream-source: ghcr.io/canonical/sdcore-nms:0.1
+    upstream-source: ghcr.io/canonical/sdcore-nms:0.2.0
 
 assumes:
   - k8s-api
 
+storage:
+  config:
+    type: filesystem
+    minimum-size: 5M
+
 requires:
   fiveg_n4:
     interface: fiveg_n4
```

Comment:
fiveg_n4 relation is no longer limit to 1. It can be removed

Reply:
Done